packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -62,7 +62,7 @@ const Redirect = ({
             break;
         }
         case 'verification': {
-            sessionStorage.setItem('redirect_url', `${routes.cashier_p2p}#verification`);
+            sessionStorage.setItem('redirect_url', routes.cashier_p2p_verification);
```

Comment:
```suggestion
            // Removing this will break mobile DP2P app. Do not remove.
            sessionStorage.setItem('redirect_url', routes.cashier_p2p_verification);
```